native/src/components/EventsDateFilter.tsx

```diff
@@ -65,6 +65,9 @@ const EventsDateFilter = ({
   const { t } = useTranslation('events')
   const currentInput = useRef<string>('from')
 
+  const today = DateTime.now()
+  const inAWeek = DateTime.now().plus({ days: 7 })
```

Comment:
🔧 I think this is simpler and also works in calendars that have different week lengths (not sure if such a calendar actually exists) .
```suggestion
  const inAWeek = DateTime.now().plus({ weeks: 1 })
```

Reply:
I am very curious to see if there is another week length somewhere in the world! 😄  But I agree, Ill make the change.

Reply:
Let me know if you find out 😝